src/main/java/com/minecolonies/coremod/entity/ai/citizen/lumberjack/EntityAIWorkLumberjack.java

```diff
@@ -227,6 +235,17 @@ protected IAIState decide()
         return GET_RECIPE;
     }
 
+    @Override
```

Comment:
see my comment above, this shouldnt be here at all

Reply:
Done